File: Java/Prime.java
public class Prime{
    public static void main(String [] args)
    {
        int num=2;
        int count=0;

        if(num<=1)
        {
            System.out.println("Not Prime");
        }

        for(int i=1;i<=num/2;i++)
        {
            if(num%i==0)
            {
                count++;
            }
        }

        if(count>1)
        {
            System.out.println("Prime Number");
        }
        else
        {
            System.out.println("Not Prime");
        }
    }
}

File: NodeJS/Prime.js
function isPrime(num)
{
    if(num<=1)
    {
        return false;
    }

    for(let i=2;i<=(num/2);i++)
    {
        if(num%i==0)
        {
            return false;
        }
    }
    return true;
}

const input_integer=79

if(isPrime(input_integer))
{
    console.log("Prime Number");
}
else
{
    console.log("Not a Prime Number");
}